frontend/src/components/campAdmin/DialogPeriodForm.vue

```diff
@@ -11,12 +11,14 @@
       :name="$tc('entity.period.fields.start')"
       vee-id="start"
       vee-rules="required"
+      :max="localPeriod.end"
```

Comment:
This one depends a bit on how we progress in https://github.com/ecamp/ecamp3/pull/3127/
Depending on the edit mode this can/cannot make sense

Reply:
Absolutely. I just wanted to leave the code in a consistent state, no matter what the decision in #3127 comes out to be. I didn't want to introduce client-side validation on period create, but not on update.